run-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This script is an opinion and I'm happy to remove it, however I struggled with figuring out how to set the environment variables correctly. *Especially* the one for sRTM net.

Reply:
This is a bit tricky with the emulator path...the newer emulators work off an hdf5 file to help mitigate this confusion, but for the old directory-style versions this is awkward.  The solution looks correct though.